pkg/maps/cells.go

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package maps
+
+import (
+	"github.com/cilium/cilium/pkg/hive/cell"
+	"github.com/cilium/cilium/pkg/maps/authmap"
+	"github.com/cilium/cilium/pkg/maps/configmap"
+)
+
+// Cell contains all cells which are providing BPF Maps.
+var Cell = cell.Module(
```

Comment:
When should I add something here vs. to the datapath cell as done for the configmap?

Reply:
Ah nvm utime.Cell is not a map.

Reply:
exactly - utime just syncs the utime and uses the configmap to write from user- to kernelspace. 

the cell of the configmap itself is part of the maps cell.